save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/agent/AgentEnvName.kt

```diff
@@ -4,18 +4,12 @@ package com.saveourtool.save.agent
  * Env names which agent supports and expects
  */
 enum class AgentEnvName {
-    ADDITIONAL_FILES_LIST,
     AGENT_ID,
     BACKEND_URL,
-    BATCH_SEPARATOR,
```

Comment:
whyyyyyyyyy removed

Reply:
it was moved to `com.saveourtool.save.agent.SaveCliOverrides` and provided in scope of the first response